src/System.Net.Security/tests/FunctionalTests/NegotiateStreamTestForUnix.cs

```diff
@@ -46,7 +47,7 @@ public KDCSetup()
                         Dispose();
                     }
 
-                    Assert.True(false, "KDC setup failure");
+                    _successfulSetup = false; // TODO: https://github.com/dotnet/corefx/issues/12107
```

Comment:
We do not need this TODO in either master or rel/1.1.0, but it was part of Steve's commit so it came along. I didn't want to add another commit just to remove it.

By that same logic, I'll remove it in master in my next PR.